Add the step to enable logging for the registry

mamidipalli opened the pull request:
By default, an IoT Core Registry's Stackdriver Logging is set to None. In order to be able to receive MQTT logs, the logging needs be enabled. This step is missing in the Readme.md, hence adding.

mamidipalli committed:
Add the step to enable logging for the registry

By default, an IoT Core Registry's Stackdriver Logging is set to None. In order to be able to receive MQTT logs, the logging needs be enabled. This step is missing in the Readme.md, hence adding.

devunwired commented on README.md:
```diff
@@ -100,6 +100,11 @@ state change events from that device. You can use the following steps to export
 which will report connection state changes.
 
+1. Enable logging for IoT Core Registry.
+   *  Navigate to IoT Core Registries page in the Cloud Console.
```
[IoT Core Registries page] should be a link to https://console.cloud.google.com/iot/registries to be consistent with the rest of the README.

mamidipalli replied:
I agree with you on being consistent Dave, but https://console.cloud.google.com/iot/registries is not opening the intended page. The link to go to the Registries page directly has Project ID in it. Example: https://console.cloud.google.com/iot/registries?project=my-gcp-project. Otherwise, it is going to *Google Cloud IoT API* page. This happens even if you have an active session to GCP available. I thought it might confuse. 
Please let me know your thoughts.

devunwired replied:
Alright, we'll keep this as-is.

mamidipalli replied:
Sure. Thank you.

devunwired commented on README.md:
```diff
@@ -100,6 +100,11 @@ state change events from that device. You can use the following steps to export
+1. Enable logging for IoT Core Registry.
+   *  Navigate to IoT Core Registries page in the Cloud Console.
+   *  Click the ID of the Registry, and then click **EDIT REGISTRY** at the top of the page.
+   *  Under **Stackdriver Logging**, select a log level.
```
Looks like connect/disconnect are only logged at INFO level, so we should probably be explicit about making that choice here.

mamidipalli replied:
Yes, Dave, I agree. I will update.

devunwired reviewed (changes requested):
Thanks for noticing this. I agree this is a good change. Just a couple minor comments.

mamidipalli committed:
Specify the log level to be selected

Looks like connect/disconnect are only logged at INFO level, so we are being explicit about making that choice here.
